kentik_api_library/pyproject.toml

```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["setuptools", "wheel", "setuptools_scm", "gitpython"]
+requires = ["setuptools_scm"]
```

Comment:
Isn't `wheel` actually needed?

Reply:
Hmm it builds without this dependency but seems to use some fallback instead; restoring